api/README.md

```diff
@@ -224,10 +228,209 @@ Build with Cloud Build and TAG:
 **Note**: Cloud Build needs to run from parent directory for build context and the [shared](../shared) directory
 
     cd ../../
-    gcloud builds submit --config api/v1/cloudbuild.yaml --substitutions=TAG_NAME=${TAG}
+    gcloud builds submit --config api/v1/api-cloudbuild.yaml --substitutions=TAG_NAME=${TAG}
 
 [Enable the APIs](https://console.cloud.google.com/flows/enableapi?apiid=cloudapis.googleapis.com,container.googleapis.com,run.googleapis.com) before beginning
 
+
+### Deploy Cloud Run Managed
+
+Deploy with Cloud Run (Managed): \
+**Note**: There are a few environment variables that need to be set before the application starts (see below). [gcloud run deploy](https://cloud.google.com/sdk/gcloud/reference/run/deploy#--set-env-vars) provides details for how they are set.
+
+* Deploy the service
+* Setup API Gateway
+* Create API Gateway Config
+* Create the API Gateway
+* Map custom domain to the service (optional)
+
+Set REGION and ZONE environmental variables
+
+    REGION=us-central1
+    ZONE=us-central1-a
+    gcloud config set compute/zone $ZONE
+
+#### Deploy the service
+
+Deploy the DS API service in Cloud Run (managed)
+
+    gcloud run deploy ds-api \
+        --image gcr.io/${PROJECT_ID}/ds-api:${TAG} \
+        --region=$REGION \
+        --no-allow-unauthenticated \
+        --platform managed \
+        --service-account ${SERVICE_ACCOUNT_NAME}@${PROJECT_ID}.iam.gserviceaccount.com \
+        --update-env-vars=OAUTH_CLIENT_ID=${OAUTH_CLIENT_ID},DATA_PRODUCERS=${DATA_PRODUCERS} \
+        --remove-env-vars=PROJECT_ID,MARKETPLACE_INTEGRATION
```

Comment:
We can eliminate the ```--remove-env-vars``` this is just for upgrades.

Reply:
ack